store/postgres/src/relational.rs

```diff
@@ -500,34 +505,39 @@ impl Layout {
     pub fn find_many(
```

Comment:
maybe comment on what is going on in this function after the changes could be helpful